src/components/shared/eventcard/eventcard.js

```diff
@@ -1,51 +1,51 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { parse, format } from 'date-fns';
 
 import Day from 'components/shared/day';
 import Box from 'components/shared/box';
 import Text from 'components/shared/text';
 import Button from 'components/shared/button';
+import VenueLink from 'components/shared/venuelink';
 
 import EventPlaceholderImage from 'images/event-placeholder.png';
-
 import ClockIcon from 'images/clock.svg';
 import MapMarkerIcon from 'images/map-marker.svg';
 
-import VenueLink from 'components/shared/venuelink';
-
 import {
   Article,
   Description,
   FeaturedImageWrapper,
   ImageWrapper,
   Image,
   RSVPBox,
-  IconList,
   IconRow,
   IconText,
 } from './eventcard.css';
 
-import { parse, format } from 'date-fns';
-
 const EventCard = ({ event, featured }) => {
   const {
+    featured_photo,
+    local_date,
+    local_time,
     name,
     plain_text_description,
-    local_date,
-    time,
-    featured_photo,
-    venue,
     short_link,
+    venue,
   } = event;
 
   const [short_description] = plain_text_description.split('\n');
-  const parsedDateTime = parse(time, 'T', new Date());
+  const parsedDate = parse(local_date, 'yyyy-MM-dd', new Date());
+  const parsedDateTime = parse(local_time, 'HH:mm', parsedDate);
   const formattedTime = format(parsedDateTime, 'h:mm a');
 
   return (
     <Article>
       <Box p={32} pb={0} gridArea="date">
-        <Day date={local_date} alignItems={{ _: 'start', sm: 'center' }} />
+        <Day
+          date={parsedDateTime.toISOString()}
```

Comment:
@timhwang21 This should be `date={parsedDate.toISOString()}`.

Reply:
Shouldn't be a difference here!

Reply:
Uh, yep, you're right. Please disregard.

Reply:
Do you need to go through the effort of parsing and converting `local_date`? The meetup API delivers that field to you in ISO 8601 format.

Reply:
Oh good call out!